### PR TITLE
Update Microbit v2 README

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

Cleans up the section on installing apps and correctly orders arguments to Tockloader so the command works.

Also fixes small spelling/grammar issues.

### Testing Strategy

Tested on hardware.

### TODO or Help Wanted

Waiting on the resolution of https://github.com/tock/tockloader/pull/72. We might want to change the recommended argument to be a minimum app size instead.

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Review by hudson-ayers — APPROVED** [review]

File: boards/microbit_v2/README.md [hunk 1]
```diff
@@ -112,21 +100,36 @@ $ make flash
...
 ```
 
-> `--bundle-apps` seems to be needed due to an [openocd issue](https://github.com/tock/tockloader/issues/67)
+> `--bundle-apps` seems to be needed due to an [openocd issue](https://github.com/tock/tockloader/issues/67).
+
+> If you are using an older version of Tockloader, add `--page-size 512` at the end of the command.
+>
+> ```bash
+> $ tockloader ... --page-size 512
+> ```
```
Comment (brghena) [inline boards/microbit_v2/README.md:132]: @bradjc Is there a way to manually specify minimum app size for Tockloader? Or should I leave `--bundle-apps` here? It matters for people not on the newest Tockloader update.

Comment (bradjc) [reply boards/microbit_v2/README.md:132]: No.

**Review by ppannuto — APPROVED** [review]
bors d+

I think this should be good to go with the tockloader update

**Pull request merged**